### ISSUE TITLE
Document NSAllowsArbitraryLoadsInWebContent

### ISSUE THREAD

**Issue by jamonholmgren** [react-native-community/react-native-webview#248]
There's a permission key in iOS called `NSAllowsArbitraryLoadsInWebContent` which generally speaking should be added to your iOS project if you're using RN WebView. Let's document that.

**Comment by jamonholmgren** [comment]
Via @ryanlntn: 

```
<key>api.merchante-solutions.com</key>
<dict>
  <key>NSExceptionRequiresForwardSecrecy</key>
  <false/>
</dict>
```

According to Apple’s docs this won’t trigger any issues with the App Store.

**Comment by MateusAndrade** [comment]
You can find more about this key here: https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloadsinwebcontent

**Comment by jamonholmgren** [comment]
From what I understand, you're not able to load http (insecure) web pages without this key. I could be wrong.

**Comment by MateusAndrade** [comment]
> From what I understand, you're not able to load http (insecure) web pages without this key. I could be wrong.

Yes! You are right, its a common mistake on IOS. I see on some places that Android 9 will need a similar config to enable HTTP calls
